test_zeroconf.py

```diff
@@ -968,3 +1074,8 @@ def send(out, addr=r._MDNS_ADDR, port=r._MDNS_PORT):
         assert service_removed.is_set()
         browser.cancel()
         zeroconf_browser.close()
+
+# Support for running outside of test tool
+# test = ListenerTest()
+# test.test_integration_with_listener_class()
+# test.test_integration_with_listener_class_with_update()
```

Comment:
This is out of scope of this patch, I'd remove.

Reply:
OK